$OpenBSD: patch-libdivecomputer_src_Makefile_am,v 1.1.1.1 2018/08/14 17:18:22 schwarze Exp $
Remove obvious third-party files aes.c and aes.h
that have no author information and no license.

Index: libdivecomputer/src/Makefile.am
--- libdivecomputer/src/Makefile.am.orig
+++ libdivecomputer/src/Makefile.am
@@ -52,7 +52,6 @@ libdivecomputer_la_SOURCES = \
 	hw_ostc.h hw_ostc.c hw_ostc_parser.c \
 	hw_frog.h hw_frog.c \
 	hw_ostc3.h hw_ostc3.c \
-	aes.h aes.c \
 	cressi_edy.h cressi_edy.c cressi_edy_parser.c \
 	cressi_leonardo.h cressi_leonardo.c cressi_leonardo_parser.c \
 	zeagle_n2ition3.h zeagle_n2ition3.c \
